tools/KoreBuild.Console/Commands/DockerFiles/ubuntu.dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM ubuntu:16.04
```

Comment:
By the way, consider using this as your base: https://github.com/dotnet/dotnet-docker/blob/master/2.0/runtime-deps/stretch/amd64/Dockerfile

```
FROM microsoft/dotnet:2.0-runtime-deps
```